docs/SOPs/GDPR_Guidelines.md

```diff
@@ -18,6 +18,18 @@ parent: SOPs
 
 This document will serve to display our guidelines for GDPR and data protection restrictions when wrangling. 
 
+## Summary diagram
+
 <iframe width="768" height="432" src="https://miro.com/app/live-embed/o9J_lzQKwKI=/?moveToViewport=-2604,-1179,5558,2621" frameBorder="0" scrolling="no" allowFullScreen></iframe>
 
 ![image](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/GDPR_screenshots/GDPR_Flowchart.jpg?raw=true)
+
+## Specific guidelines
+
+GDPR concers arise when dealing with living donor organisms, as HCA sticks to a strict interpretation of the regulations. When wrangling any kind of data related to living donors, wranglers can only ingest into the system data that is already in the public domain.
+
+When wrangling from already **published projects**, wranglers should **never ask for extra donor metadata** if it involves living donors. Wranglers are only allowed to take the information that the authors have made already of public domain. Moreover, to prevent low *k-anonymity* [^1] levels, wranglers should **avoid wrangling donor metadata when the project includes less than 5 living donors**. 
```

Comment:
I suggest making a clearer distinction between the following and in separate paragraphs:

- rules for metadata
- rules for data (fastq, gene expression matrices)

Reply:
I would say the first paragraph is referring to all kinds of data, as being strict and until discussed with Tony we are not able to wrangle any kind of data (including metadata) if it's not already in the public domain. 

The second paragraph is making an emphasis on metadata

Reply:
Ah ok, I missed some context. I spoke with @tburdett earlier today on the metadata slack channel and he said he would need to review the "all kinds of data", specifically in response to my question about gene expression matrices. I am happy to leave this as it is if the "all kinds of data" is confirmed. However, I am not fully convinced we shouldn't add the matrices in this case, until Tony has got back on this.